apps/ledger-live-mobile/e2e/specs/manager.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { expect, device } from "detox";
+import { loadBleState, loadConfig } from "../bridge/server";
+import PortfolioPage from "../models/wallet/portfolioPage";
+import DeviceAction from "../models/DeviceAction";
+import ManagerPage from "../models/manager/managerPage";
+import { DeviceModelId } from "@ledgerhq/devices";
+import { getElementByText, waitForElementByText } from "../helpers";
+
+let portfolioPage: PortfolioPage;
+let deviceAction: DeviceAction;
+let managerPage: ManagerPage;
+
+const knownDevice = {
+  name: "Nano X de test",
+  id: "mock_1",
+  modelId: DeviceModelId.nanoX,
+};
+
+describe("Bitcoin Account", () => {
+  beforeAll(async () => {
+    loadConfig("onboardingcompleted", true);
+    loadBleState({ knownDevices: [knownDevice] });
+
+    portfolioPage = new PortfolioPage();
+    deviceAction = new DeviceAction(knownDevice);
+    managerPage = new ManagerPage();
+
+    await portfolioPage.waitForPortfolioPageToLoad();
+  });
+
```

Comment:
I would add an afterAll reenabling the detox synchronisation just to make sure it doesn't affect the other tests

Reply:
it doesn't. i think this settings is protected and have no side effect outside of the scope where you run it (if you run it inside a `it` it starts local to that `it` test)